src/scratchtocatrobat/converter/test_converter.py

```diff
@@ -2153,12 +2153,26 @@ def test_can_convert_call_block_user_script_not_yet_defined_simple(self):
         assert expected_param_types == param_types
 
     #gotoSpriteOrMouse:
-    def test_can_convert_go_to_sprite_block_with_sprite(self):
-        test_sprite_name = "Abby"
+    def test_can_convert_go_to_sprite_block_with_sprite_afterwards(self):
+        test_sprite_name = "Afterwards"
         scratch_block = ["gotoSpriteOrMouse:", test_sprite_name]
+        context = converter.Context()
+        sprite_context = converter.SpriteContext(DUMMY_CATR_SPRITE.getName(), {})
+        sprite_context.context = context
+        script_context = converter.ScriptContext(sprite_context)
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE, script_context)
+        assert isinstance(catr_brick, catbricks.GoToBrick)
+        assert catr_brick.destinationSprite.getName() == test_sprite_name
```

Comment:
I'd suggest to add some additional checks here to ensure that Context object gets populated with the reference pointing to the same Sprite object, e.g.:
```
        assert "Afterwards" in context.upcoming_sprites
        assert catr_brick.destinationSprite is context.upcoming_sprites["Afterwards"]
```